src/Scaffold_Command.php

```diff
@@ -201,6 +200,106 @@ private function _scaffold( $slug, $assoc_args, $defaults, $subdir, $templates )
 		}
 	}
 
+	/**
+	 * Generate PHP, JS and CSS code for registering a custom block.
+	 *
```

Comment:
Would this benefit from a longer explanation of what a block is, etc.?

Reply:
Added.